src/applications/personalization/profile360/components/ProfileView.jsx

```diff
@@ -20,18 +20,19 @@ import PaymentInformation from '../containers/PaymentInformation';
 import IdentityVerification from './IdentityVerification';
 import MVIError from './MVIError';
 
-const ProfileTOC = ({ militaryInformation }) => (
+const ProfileTOC = ({ militaryInformation, isDirectDepositEligible }) => (
   <>
     <h2 className="vads-u-font-size--h3">On this page</h2>
     <ul>
       <li>
         <a href="#contact-information">Contact information</a>
       </li>
-      {!environment.isProduction() && (
-        <li>
-          <a href="#direct-deposit">Direct deposit information</a>
-        </li>
-      )}
+      {!environment.isProduction() &&
+        isDirectDepositEligible && (
+          <li>
+            <a href="#direct-deposit">Direct deposit information</a>
+          </li>
+        )}
```

Comment:
I wonder if it would have been simpler or kept the DD logic abstracted by exporting this link as a container (maybe an additional export from `PaymentInformation`) that controls its own rendering based on user eligibility.  That way we can kinda avoid this parent component from having to deal with stuff that could be under-the-hood of another component. What do you think?

Reply:
cool, I'll noodle around with that idea!